integrations/access/servicenow/client.go

```diff
@@ -90,10 +91,11 @@ func NewClient(conf ClientConfig) (*Client, error) {
 		return nil, trace.Wrap(err)
 	}
 
-	if apiURL.Scheme == "http" && !strings.HasPrefix(apiURL.Host, "127.0.0.1") {
+	hostNoPort, _, err := net.SplitHostPort(apiURL.Host)
```

Comment:
You are not checking the `err` value.
But you can use `apiURL.Hostname()` to get the same result

Reply:
Oh nice. Didn't know that existed.